### PR TITLE
updating README

### PR DESCRIPTION
**Pull request by jvanbrie**
*Note: Please remember to review the [contribution guidelines](https://github.com/DataDog/cloudformation-template/blob/master/CONTRIBUTING.md)
if you have not yet done so.*

### What does this PR do?
Updates the quickstart README. It now has up-to-date instructions for using the template, and instructions for updating the created CloudFormation template.

### CODE REVIEW

File: aws_quickstart/release.sh [hunk 1]
```diff
@@ -55,7 +55,7 @@ cp main_extended.yaml main_extended.yaml.bak
 perl -pi -e "s/<BUCKET_PLACEHOLDER>/${BUCKET}/g" main_extended.yaml
 perl -pi -e "s/<VERSION_PLACEHOLDER>/${VERSION}/g" main_extended.yaml
 
-trap 'mv main.yaml.bak main.yaml; mv main_v2.yaml.bak main_v2.yaml; mv main_extended.yaml.bak main_extended.yaml' EXIT
+trap 'mv main_v2.yaml.bak main_v2.yaml; mv main_extended.yaml.bak main_extended.yaml' EXIT
```
Comment (jvanbrie) [inline aws_quickstart/release.sh:58]: Drive-by fix to the release script cleanup.

File: aws_quickstart/README.md [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
-   1. Enter IAM Role name `DatadogIntegrationRole` (needs to match the value of `IAMRoleName` in the next step).
-   1. Copy the External ID for the next step to use.
-1. Log into your admin AWS account/role and deploy the CloudFormation Stack with the button above.
-   1. Fill in all the `Required` parameters.
-   1. Optinally edit `LogArchives` and `CloudTrails` to configure [Log Archives](https://docs.datadoghq.com/logs/archives/?tab=awss3) and [CloudTrail](https://docs.datadoghq.com/integrations/amazon_cloudtrail/) integration.
-   1. On a rare occasion, if you already have a stack deployed in the same AWS account using this template (e.g., monitor the same AWS account in multiple Datadog accounts), You MUST use a different role name for `IAMRoleName` and set `InstallDatadogPolicyMacro` to `false`.
+   1. Click "Add AWS Account(s)".
+   1. Fill out the various fields under "Automatically using CloudFormation".
+   1. Click "Launch CloudFormation Template" in the bottom right.
+1. Log into your admin AWS account/role and deploy the CloudFormation Stack that you're linked to. All required paramaters will be filled in.
```
Comment (ktmq) [inline aws_quickstart/README.md:11]:
```suggestion
1. Log into your admin AWS account/role and deploy the CloudFormation Stack that you're linked to. All required parameters will be filled in.
```

File: aws_quickstart/README.md [hunk 3]
```diff
@@ -1,18 +1,17 @@
 # Datadog AWS Integration
 
-[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?stackName=datadog&templateURL=https://datadog-cloudformation-template.s3.amazonaws.com/aws/main_v2.yaml)
+[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?stackName=datadog&templateURL=https://datadog-cloudformation-template.s3.amazonaws.com/aws/v2.0.6/main_v2.yaml)
```
[the remark this answers is not in the document]
Comment (jvanbrie) [reply aws_quickstart/README.md:3]: Thanks for the catch, updated.

File: aws_quickstart/README.md [hunk 4]
```diff
@@ -1,18 +1,17 @@
...
-   1. Copy the External ID for the next step to use.
-1. Log into your admin AWS account/role and deploy the CloudFormation Stack with the button above.
-   1. Fill in all the `Required` parameters.
-   1. Optinally edit `LogArchives` and `CloudTrails` to configure [Log Archives](https://docs.datadoghq.com/logs/archives/?tab=awss3) and [CloudTrail](https://docs.datadoghq.com/integrations/amazon_cloudtrail/) integration.
-   1. On a rare occasion, if you already have a stack deployed in the same AWS account using this template (e.g., monitor the same AWS account in multiple Datadog accounts), You MUST use a different role name for `IAMRoleName` and set `InstallDatadogPolicyMacro` to `false`.
+   1. Click "Add AWS Account(s)".
+   1. Fill out the various fields under "Automatically using CloudFormation".
+   1. Click "Launch CloudFormation Template" in the bottom right.
+1. Log into your admin AWS account/role and deploy the CloudFormation Stack that you're linked to. All required parameters will be filled in.
+   1. Optinally edit "Advanced" fields to set the IAM RoleName used, and disable default collection settings.
```
Comment (ktmq) [inline aws_quickstart/README.md:12]:
```suggestion
   1. Optionally edit "Advanced" fields to set the `IAMRoleName` used, or disable default collection settings.
```

**Review by ktmq — APPROVED** [review]
